### DISCUSSION TITLE
Disable "jumping" to the next comment

### DISCUSSION THREAD

**Discussion by NinovanderMark** [community/community#8200] [Mobile]
When I'm scrolling through the comments on an issue, the mobile app seems to jump across parts of the comment, causing me to be disoriented about what I have and haven't seen or read.

Please provide a way to turn this off, thank you.

**Comment by eliperkins** [comment]
Thanks for the feedback @NinovanderMark! Jumping around on comments definitely sounds disorienting. Can I get some more information from you to help debug here?

- What platform are you experiencing this on (iOS or Android)? What version of the app as well?
- When do you experience this? Is it only when scrolling or does it happen when navigating to an issue/pull request from a notification?

I'd love to get this addressed, so the more info I can use to reproduce this and bring to our engineering team, the better!

**Reply by NinovanderMark** [reply]
I'm running Android 10, and using the GitHub version from Oct 12th. I was actually under the impression it was some sort of accessibility feature. 😅

I've just tried to reproduce the issue, but I'm unable to at this time. 

I understand this is a pretty poor bug report, so feel free to close it if no-one else has run into it.
